vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/project/samples/batchRegistration/SampleSpreadsheetLayout.java

```diff
@@ -107,83 +169,303 @@ private static class SampleRegistrationSheetBuilder {
+      headerToPresets.put("Condition", conditionsToReplicates.keySet().stream().toList());
+      headerToPresets.put("Biological replicate id", getReplicateLabels());
+      prepareColumnHeaderAndWidth(spreadsheet, headerToPresets);
       spreadsheet.reload();
+      setupCommonDropDownColumns();
     }
 
-    private void addMetabolomicsSheet(Spreadsheet spreadsheet) {
-      setAndStyleHeader(spreadsheet, sampleRegistrationService.retrieveMetabolomics());
-      spreadsheet.reload();
+    private void addProteomicsSheet(Spreadsheet spreadsheet, List<String> header) {
+      this.header = header;
+      prepareCommonSheetTasks(spreadsheet);
+
     }
 
-    private void addLigandomicsSheet(Spreadsheet spreadsheet) {
-      setAndStyleHeader(spreadsheet, sampleRegistrationService.retrieveLigandomics());
-      spreadsheet.reload();
+    private void addMetabolomicsSheet(Spreadsheet spreadsheet, List<String> header) {
+      this.header = header;
+      prepareCommonSheetTasks(spreadsheet);
+    }
+
+    private void addLigandomicsSheet(Spreadsheet spreadsheet, List<String> header) {
+      this.header = header;
+      prepareCommonSheetTasks(spreadsheet);
     }
 
-    private void addGenomicsSheet(Spreadsheet spreadsheet) {
-      setAndStyleHeader(spreadsheet, sampleRegistrationService.retrieveGenomics());
+    private void addGenomicsSheet(Spreadsheet spreadsheet, List<String> header) {
+      this.header = header;
+      LinkedHashMap<String, List<String>> headerToPresets = new LinkedHashMap<>();
+      for(String head : header) {
+        headerToPresets.put(head, new ArrayList<>());
+      }
+      headerToPresets.put("Species", species);
+      headerToPresets.put("Specimen", specimens);
+      headerToPresets.put("Condition", conditionsToReplicates.keySet().stream().toList());
+      headerToPresets.put("Analysis to be performed", Arrays.stream(SequenceAnalysisTypes
+              .values())
+          .map(e -> e.label)
+          .collect(Collectors.toList()));
+      headerToPresets.put("Biological replicate id", getReplicateLabels());
+      prepareColumnHeaderAndWidth(spreadsheet, headerToPresets);
       spreadsheet.reload();
-      SpreadsheetDropdownFactory dropdownCellFactory = new SpreadsheetDropdownFactory();
-      //TODO this should be known from experimental groups and sample size
-      int maximumNumberOfSamples = 100;
-      int firstDataRow = 1;
-      int dropDownColumn = 0;
-      unlockColumn(spreadsheet, dropDownColumn, firstDataRow, maximumNumberOfSamples+firstDataRow);
-      dropdownCellFactory.fromColIndex(dropDownColumn).toColIndex(dropDownColumn);
-      dropdownCellFactory.fromRowIndex(1).toRowIndex(maximumNumberOfSamples+firstDataRow);
-      dropdownCellFactory.withItems(Arrays.asList("DNA-Seq", "RNA-Seq"));
-      spreadsheet.setSpreadsheetComponentFactory(dropdownCellFactory);
+      DropDownColumn techColumn = new DropDownColumn().withItems(Arrays.stream(SequenceAnalysisTypes
```

Comment:
NIT: consistent naming since the cells within this column are called analysisTypeCell 
```suggestion
      DropDownColumn analysisTypeColumn = new DropDownColumn().withItems(Arrays.stream(SequenceAnalysisTypes
```